src/commands/case_management.py

```diff
@@ -143,8 +149,16 @@ async def cmd_case_management_active(ctx: Context):
         return
 
     # We either have a valid case or we've left the method at this point.
-    async with ctx.bot.board.modify_rescue(rescue) as case:
+    async with ctx.bot.board.modify_rescue(rescue, impersonation=ctx.user.account) as case:
+        logger.debug(f"Switching case to active = {not case.active}")
         case.active = not case.active
+        logger.debug(f"Inject message: {tokens.remainder}")
+        if tokens.remainder:
+            # Inject message before toggling active/inactive if it is passed
+            case.add_quote(ctx.words_eol[0], ctx.user.nickname)
+            await ctx.reply(
+                f"{case.client}'s case updated with: {ctx.words_eol[2]!r} (Case {case.board_index})"
```

Comment:
[error] You inject `ctx.words_eol[0]` but indicate you update with `ctx.words_eol[2]`.
```suggestion
                f"{case.client}'s case updated with: {ctx.words_eol[0]!r} (Case {case.board_index})"
```